api/v1alpha1/common_types.go

```diff
@@ -49,9 +49,9 @@ type InitClusterNode bool
 type PKIBackend string
 
 // ClientConfigType represents an interface implementing the ClientConfigManager
-type ClientConfigType string
 
 // ClusterType represents an interface implementing the  ClientConfigManager
+type ClientConfigType string
```

Comment:
A reason for this change ?

Reply:
Purely accidental. I've reverted this. My apologies